bin/varnishd/mgt/mgt_child.c

```diff
@@ -603,7 +603,7 @@ MGT_Child_Cli_Fail(void)
 		return;
 	if (kill_child() == 0)
 		MGT_complain(C_ERR, "Child (%jd) not responding to CLI,"
-		    " killing it.", (intmax_t)child_pid);
+		    " killed it.", (intmax_t)child_pid);
```

Comment:
Do we need an extra commit to fix a message introduced by the same pull request? I would happily squash this change in efcb76f where the message first appeared.